.github/workflows/sync_tx.yml

```diff
@@ -11,27 +11,22 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Check out repository
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
         with:
           ref: dev
           fetch-depth: 0 # otherwise, there would be errors pushing refs to the destination repository.
       - name: Setup Python
-        uses: actions/setup-python@v1
+        uses: actions/setup-python@v5
         with:
-          python-version: 3.9
-          
-      - name: Run script if manual dispatch
-        if: github.event_name == 'workflow_dispatch'
```

Comment:
this one should stay, it is useful when translation sync fails  (eg, network issues)